extension/plugins/content/SlowYTVids.js

```diff
@@ -0,0 +1,40 @@
+import ContentPlugin from '../../lib/ContentPlugin.js';
+
+class PageBlur extends ContentPlugin {
+  constructor() {
+    super('Slow down YouTube video payback speed');
+  }
+
+  id = 57;
+
+  settings = {
+    enabled: false,
+    // TODO: Maybe enable to all websites?
+    yt_only: true,
+    speed: .4,
+  }
+
+  run() {
+    const slowVID = () => {
+      document.querySelectorAll('video').forEach(vid => {
+        vid.playbackRate = (typeof speed === 'undefined') ? .4 : speed;

```

Comment:
I think you should use `this.settings.speed` here instead of just `speed`

Reply:
yeah, you're right.